spec/hangman_spec.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require './lib/hangman'
+
+RSpec.describe Hangman do
+  describe ".start" do
```

Comment:
Normally this is the method under test (sometimes the `subject`; or called on the `subject`), but it's not used in the test.

Reply:
If, as this `describe` text suggests, the goal of this PR is to fulfil the first user story:

> As a player, I want to start a game of Hangman

Then we should define a "done-when" for it. This BDD/TDD approach is great because it forces/encourages us to be outcome-oriented and focussed, rather than diving into code (somewhat randomly, as we developers are prone to do!). 

In this case, at least for me, it's hard to decide what this done-when looks like without taking a step back to see the bigger picture (where does `Hangman.start` get used?; what does the rest of the script look like?)

As suggested earlier, IMO scripts should be as simple as possible (ie leave the heavy lifting and logic to testable objects in the codebase). with this in mind the script _could_ simply invoke `Hangman.start`. But that leaves us with the complexity of testing keyboard interactions. Let's discuss some alternative approaches.